dev-libs/wolfssl/wolfssl-3.15.3.recipe

```diff
@@ -0,0 +1,92 @@
+SUMMARY="An embedded SSL implementation"
+DESCRIPTION="The wolfSSL embedded SSL library (formerly CyaSSL) is a \
+lightweight SSL/TLS library written in ANSI C and targeted for embedded, \
+RTOS, and resource-constrained environments - primarily because of its \
+small size, speed, and feature set."
+HOMEPAGE="https://www.wolfssl.com/"
+COPYRIGHT="2006-2018 Sawtooth Consulting Ltd."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/wolfSSL/wolfssl/archive/v$portVersion-stable.tar.gz"
+CHECKSUM_SHA256="2b98d9ddac2a3188210de720051bed58f91910ee028a4b0a6dd6fd3c9ddb6fec"
+SOURCE_DIR="wolfssl-$portVersion-stable"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Has the package been tested on any architecture yet?

Reply:
it builds ok on x86 (secondaryArch), need to check x86_64
the configuration files look ok and produce the correct paths (don't know yet how to check this ssl version with another source file though)